### PR TITLE
use Terraform based aws runner with autoscaling

### PR DESCRIPTION
**Pull request by mashehu**
Based on https://github.com/philips-labs/terraform-aws-github-runner

### CODE REVIEW

**Comment by Emiller88** [comment]
Looks good to me! Would you want to roll this into the ops repo and we can have directories for the different pieces? `action-runners` `github` `awsmegatests`

**Comment by mashehu** [comment]
yeah, could probably add this. I wanted this PR mainly to document the current setup

File: lambdas-download/webhook.zip
Comment (Emiller88) [inline lambdas-download/webhook.zip:1]: Are these TF modules or neccesary?

File: lambdas-download/terraform.tfstate
Comment (Emiller88) [inline lambdas-download/terraform.tfstate:1]: Another state file

**Pull request closed without merging**